### ISSUE TITLE
Unix & Unix-like newline errors

### ISSUE THREAD

**Issue by logankeenan** [brigade/scss-lint#203]
A Unix or Unix-like system will not throw a error given the example code below which should not have a space after the a tag.

``` css
a
{
    color: #000;
}
```

It will only throw an error if the rule has a space after the a tag.  A windows system works as expected and will throw an error in either situation.
